bode/bode/resources/tasks/schemas.py

```diff
@@ -22,4 +23,7 @@ class TaskSchema(BaseSchema):
     description = fields.String()
     due_date = fields.DateTime()
     status = fields.String()
+    is_blocked = fields.Function(
+        lambda obj: obj.status == TaskStatus.TODO.value and TaskRelation.is_task_blocked(obj.id)
```

Comment:
We should allow to be blocked tasks that are indirectly done as well.
```suggestion
        lambda task: task.status != TaskStatus.DONE.value and TaskRelation.is_task_blocked(task.id)
```